chicken-sloths/bangazon-site-sprint3
# Display latest products on homepage

# Description
Displays latest products when you view the Bangazon homepage.

## Related Ticket(s)
Fixes #10 

## Proposed Changes
- Renamed `productTypeCtrl` to `homepageCtrl`
- Combined functions to render product type lists and latest products list into a `displayHomePage` function

## Expected Behavior && Steps to Test
1. Run `npm run db:gen`
1. Run `npm start`
1. Log in with any user
1. The homepage should display five products above the category list. (Plot twist: they were all created at the exact same time, because you just ran `db:gen`. All products are latest products! If everyone is super no one is super! Muahahauauaha)
1. Go create a new product.
1. Go back to the homepage. You should see the product you just created at the top!
